### DISCUSSION TITLE
建议发布win64的rime_deployer.exe

### DISCUSSION THREAD

**Discussion by jack6th** [rime/home#1074] [General]
目前Windows上的小狼毫是32位的，无法部署大词库。
我制作了一个600多万的词库，Windows上无法部署。
我在MinGW64上修改librime源文件中的CMakefiles，并成功编译了64位版的rime_deployer.exe
但无法正常部署，不是大词库也无法部署，终端显示Illegal instruction，错误日志显示table.cc:316] invalid metadata.
最后切换到64位的Linux，用rime_deployer成功部署后把build文件夹中的内容在复制到Windows，这才正常使用。
希望能够发布win64的rime_deployer.exe，使用户能在Windows上部署大词库。

**Comment by jack6th** [comment]
上文所述的Illegal instruction问题，是MinGW64中64位的marisa 0.2.6-2导致的。   
这个包的维护者在编译64位版本的时候使用了--enable-native-code参数，导致只要用户的 CPU 比这位维护者使用的 CPU 旧的话，就有可能出现 Illegal instruction。（32位版本没有使用该参数，具体参见该包的 PKGBUILD 文件）   
重新编译marisa包后，rime_deployer.exe已可以正常部署。   
需要在Windows上部署超大词库的可以用MinGW64编译64位版的rime_deployer.exe，不需要借助Linux了。

**Reply by clijiac** [reply]
方便提供一下64位的rime_deployer.exe吗

**Reply by jack6th** [reply]
https://github.com/jack6th/rime_tools_win64
